hack-the-resume/src/components/Basics.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import '../styles/Basics.css'
+import logo from '../resume/assets/profile-pic.jpg'
+
+const Basics = (props) => {
+  return (
+    <div className='Basics'>
+      <img src={logo} className="Pic" alt="profile pic" />
+      <h2>{props.basics.name} | {props.basics.label}</h2>
+      <p>{props.basics.summary}</p>
+      <br/>
+      <p>{props.basics.email}</p>
+      <p>{props.basics.phone}</p>
+      <p>{props.basics.website}</p>
+      <p>{props.basics.location.address}</p>
+      <p>{props.basics.location.city}, {props.basics.location.region}</p>
+      <p>{props.basics.location.countryCode} - {props.basics.location.postalCode}</p>
```

Comment:
You could use deconstructions for each of these props so that you don't have to continue writing "props.basics". You could instead say: 

```
const {email, phone, website, name, summary} = props.basics
```
and then refer to the variables in the return statement.